docs/getting_started/install_using_ddev.md

```diff
@@ -0,0 +1,410 @@
+
+    If you wan't to use `.env.local` file instead of server-level environment variables, see [Using dotenv](#using-dotenv).
+
+=== "MariaDB / MySQL"
+
+    ```bash
+    ddev config --web-environment-add DATABASE_URL=mysql://db:db@db:3306/db
+    ```
+
+=== "PostgreSQL"
+
+    ```bash
+    ddev config --web-environment-add DATABASE_URL=postgresql://db:db@db:5432/db
+    ```
+
+#### Enable Mutagen (optional)
+
+If you're using macOS or Windows, you might wish to enable [Mutagen](https://ddev.readthedocs.io/en/latest/users/install/performance/#mutagen) for performance. You can do this by running the following command:
+
+```bash
+ddev config --mutagen-enabled
+```
+
+See [DDEV performance documentation](https://ddev.readthedocs.io/en/latest/users/install/performance/) for more.
+
+#### Change port mapping (optional)
+
+By default, DDEV uses ports 80 and 443.
+You can [set different ports](https://ddev.readthedocs.io/en/latest/users/usage/troubleshooting/#method-2-fix-port-conflicts-by-configuring-your-project-to-use-different-ports) with a command like the following:
+
+```bash
+ddev config --http-port=8080 --https-port=8443
+```
+
+### 3. Start DDEV
+
+Proceed by starting your DDEV environment with the following command:
+
+```bash
+ddev start
+```
+
+!!! tip
+
+    If you forgot some part of configuration, you can set it by using `ddev config` later, but afterwards you have to restart DDEV using `ddev restart`.
+
+### 4. Composer authentication
+
+Next, you need to [set up authentication tokens](install_ibexa_dxp.md#set-up-authentication-tokens) by modifying the Composer configuration.
+You must run the following command **after** executing  `ddev start`, because the command will run inside the container.
+
+```bash
+ddev composer config --global http-basic.updates.ibexa.co <installation-key> <token-password>
+```
+
+Replace `<installation-key>` and `<token-password>` with your actual installation key and token password, respectively.
+
+This authentication won't persist if the project is restarted (by `ddev restart` or `ddev composer create`). You can back up the authentication file (`auth.json`) using the following command:
+
+```bash
+ddev exec "mkdir -p .ddev/homeadditions/.composer; cp ~/.composer/auth.json .ddev/homeadditions/.composer"
+```
+
+If you want to reuse an existing `auth.json` file, see [Using an auth.json](#using-an-authjson).
+
+### 5. Create Ibexa DXP project
+
+Once DDEV is running, use Composer to create a new Ibexa DXP project. Remember to replace `<edition>` and `<version>` with your desired edition and versions respectively.
+
+```bash
+ddev composer create ibexa/<edition>-skeleton:<version>
+```
+
+!!! tip
+
+    You may add the Symfony Debug bundle for its var dumper enhancement which can avoid getting out of memory when dumping huge or circular variables.
+    `ddev composer require --dev symfony/debug-bundle`
```

Comment:
@adamwojs Is this introduction of the `symfony/debug-bundle` accurate?

Reply:
This is more precise statement: 

"You may add the Symfony Debug bundle for its var dumper enhancement which allows to  avoid getting out of memory error when dumping objects containing circular references."

Reply:
```suggestion
    You may add the Symfony Debug bundle for its var dumper enhancement which allows to avoid getting out of memory error when dumping objects containing circular references.
    `ddev composer require --dev symfony/debug-bundle`
```